src/oc_erchef/rebar.lock

```diff
@@ -86,7 +86,7 @@
   1},
  {<<"mini_s3">>,
   {git,"https://github.com/chef/mini_s3",
-       {ref,"8a6b80ec60be799c5f69d9b2f22eb0af55b4dc77"}},
+       {branch,"lbaker/upgrade-erlang-24.2"}},
```

Comment:
Could you please use the references If you would like to merge this to main as lock should always have the reference which doesn’t behave if you commit something into "lbaker/upgrade-erlang-24.2 branch

Reply:
See comment above.